A32NX/html_ui/Pages/A32NX_Core/A32NX_LocalVarUpdater.js

```diff
@@ -189,5 +195,24 @@ class A32NX_LocalVarUpdater {
         return this.isPacksOneSupplying;
     }
 
+    _areSlidesArmed() {
+
+        if (SimVar.GetSimVarValue('ON ANY RUNWAY', 'bool')) {
+            return true;
+        } else if (SimVar.GetSimVarValue('SIM ON GROUND', 'bool') === false) {
+            return true;
+        } else if (SimVar.GetSimVarValue('LIGHT BEACON ON', 'bool') === false) {
+            return false;
+        } else if (SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:0', 'percent') > 4) {
+            return false;
+        } else if (SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:3', 'percent') > 4) {
+            return false;
+        } else if (SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:5', 'percent') > 4) {
+            return false;
+        } else {
+            return true;
+        }
```

Comment:
- `GetSimVarValue` doesn't return an actual `boolean`, thus these expressions won't work as intended.
- You modified logic from `< 5` to `> 4` which might carry the assumption with it that 'percent' values are integers. However, they are not and thus these changes have modified the actual logic. Was this intended?
- The previous version of the code had names indicating what the various `INTERACTIVE POINT OPEN` simvars referred to. This information got lost in your change. Consider adding a comment to each line to indicate what they mean.

Consider writing:
```suggestion
        return SimVar.GetSimVarValue('ON ANY RUNWAY', 'bool') ||
            !SimVar.GetSimVarValue('SIM ON GROUND', 'bool') ||
            (SimVar.GetSimVarValue('LIGHT BEACON ON', 'bool') && 
                SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:0', 'percent') < 5 &&
                SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:3', 'percent') < 5 &&
                SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:5', 'percent') < 5
            );
```

Reply:
The code change was to prevent the need to evaluate all of the SimVarValues if one failed. I think you proposed change would require at least the evaluation of the OR statements, although I don't profess to know the inner workings of javascript. Have made other amendments based on your recommendations though.

Reply:
Logical operators short-circuit in JavaScript. In your original code version you read all SimVars before the logical expression, which was the problem @beheh was refering to.

In my suggested change, if `!SimVar.GetSimVarValue('SIM ON GROUND', 'bool')` is `true` the remainder of the expression isn't evaluated.

I also ordered based on likelihood: most of the time is spent in flight, so that condition first. ON ANY RUNWAY is unlikely but only a single condition to check, so I put that second, etc.